File: visualBasic/Glacier2/callback/CallbackReceiverI.vb
' **********************************************************************
'
' Copyright (c) 2003-2016 ZeroC, Inc. All rights reserved.
'
' **********************************************************************

Imports Demo
Imports System

Public NotInheritable Class CallbackReceiverI
    Inherits CallbackReceiverDisp_

    Public Overloads Overrides Sub callback(ByVal current As Ice.Current)
        Console.WriteLine("received callback")
    End Sub

End Class

File: java-compat/Ice/callback/CallbackReceiverI.java
// **********************************************************************
//
// Copyright (c) 2003-present ZeroC, Inc. All rights reserved.
//
// **********************************************************************

import Demo.*;

public final class CallbackReceiverI extends _CallbackReceiverDisp
{
    @Override
    public void
    callback(Ice.Current current)
    {
        System.out.println("received callback");
    }
}
